### DISCUSSION TITLE
Quasar project use Epson ePOS SDK for JavaScript to print!

### DISCUSSION THREAD

**Discussion by IvanWala** [quasarframework/quasar#10779] [General - Components / Directives / etc] [answered]
Following this topic
https://forum.quasar-framework.org/topic/5851/quasar-project-use-epson-epos-sdk-for-javascript-to-print

I need to add a meta with this external script.  

The problen is the file doesn't has a module.export and i don't know how properly add this file to a quasar project. 

Is possible import a external file without export in a quasar project trought meta plugin? And most important how can we do it? The documentation doesn't explain it.

 https://quasar.dev/quasar-plugins/meta

**Comment by metalsadman** [comment, marked as answer]
You can append it in the dom i suppose.

**Comment by ontwikkelfabriek** [comment]
@metalsadman 

> You can append it in the dom i suppose.

Do you refer to adding it to the `index.template.html` as a JS script? Because the link to the forum suggest using the `Meta` plugin, but I don't really understand how to do that. Do you?

**Reply by metalsadman** [reply]
haven't tried so myself, but i suppose you can in that file manually, or append it later in the body via code.

**Reply by IvanWala** [reply]
OK.

I add it to the index.template.html head, And It seems to work. It prints.

Now I go to try to made a boot file. 

Thanks.
